Cargo.toml

```
@@ -49,7 +49,8 @@ native = [
 
 wasm = [
     "bevy_rapier3d/wasm-bindgen",
-    "core"
+    "core",
+    "wasm-bindgen",
```

I think this syntax works here:
```suggestion
    "dep:wasm-bindgen",
```

---

I realize I could use this syntax in some other optional deps as well, hehe

---

Uu what is this syntax?